tests/interfaces/test_jax.py

```diff
@@ -338,7 +344,13 @@ def cost(a, cache):
                 qml.expval(qml.PauliZ(0))
 
             tape = qml.tape.QuantumScript.from_queue(q)
-            return execute([tape], dev, gradient_fn=param_shift, cache=cache, interface=interface,)[
+            return execute(
+                [tape],
+                dev,
+                gradient_fn=param_shift,
+                cache=cache,
+                interface=interface,
+            )[
                 0
             ][0]
```

Comment:
What's going on here? Could removing the trailing comma in the old version change the behaviour of black to not do this? Or do we prefer this?

Reply:
Removed the last comma!

Reply:
Much better now!